x-pack/test/functional/apps/canvas/lens.ts

```diff
@@ -10,10 +10,16 @@ import { FtrProviderContext } from '../../ftr_provider_context';
 export default function canvasLensTest({ getService, getPageObjects }: FtrProviderContext) {
   const PageObjects = getPageObjects(['canvas', 'common', 'header', 'lens']);
   const esArchiver = getService('esArchiver');
+  const kibanaServer = getService('kibanaServer');
+  const archives = {
+    es: 'x-pack/test/functional/es_archives/canvas/logstash_lens',
+    kbn: 'x-pack/test/functional/fixtures/kbn_archiver/canvas/lens',
+  };
```

Comment:
whats the difference between these two archives?

Reply:
A couple differences...
`archives.es` is not from a `.kibana` index, so it should be loaded with `es_archiver`. 
 - In fact, `kbn_archiver` cannot load this data, iirc.  

`archives.kbn` contains only stanzas from `.kibana` indexes, so it should be loaded with `kbn_arhiver`.  

So, one is cool, one is not? lol. Kidding.

Does that answer the question Poff?